server/routes/index.ts

```diff
@@ -17,12 +17,15 @@ import populateCurrentUser from '../middleware/populateCurrentUser'
 import flashMessages from '../middleware/flashMessageMiddleware'
 import fromReviewMiddleware from '../middleware/fromReviewMiddleware'
 import PrisonerController from './prisonerController'
+import WhatsNewPage from './whatsNewPage/whatsNewPage'
```

Comment:
There is a convention that these are called `Controller` not `Page`

(Page is a bit confusing as this is what we call e2e test page objects)

Reply:
renamed it to WhatsNewController